resources/bin/build.sh

```
@@ -0,0 +1,118 @@
+        -sdk $(xcrun --show-sdk-path) \
+        -framework EventKit \
+        -framework Foundation; then
+        echo "エラー: swiftc によるビルドが失敗しました" >&2
+        exit 1
+    fi
+
+    # Apple Silicon の場合は Universal Binary を作成
+    if [ "$(uname -m)" = "arm64" ]; then
+        echo "Universal Binary を作成しています..."
+        swiftc -o "$BUILD_DIR/${PRODUCT_NAME}_arm64" CalendarAccessTool.swift \
+            -sdk $(xcrun --show-sdk-path) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)